### PR TITLE
`lintr` options

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Hi,

I'm not really proficient in VimScript or vader, so I would greatly appreciate if someone could help me write tests for this, but I *think* that I allowed for linters to be set for `lintr` for `R` files. So far it's been working for me (informally), with the following line in my `vimrc`:
```
let·g:ale_r_lintr_options·=·'lintr::with_defaults(object_usage_linter·=·NULL)'
```

Loving ALE so far, Thanks!

### CODE REVIEW

File: ale_linters/r/lintr.vim [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
+let g:ale_r_lintr_options =
+\   get(g:, 'ale_r_lintr_options', 'lintr::with_defaults()')
+" A reasonable alternative default:
+" \   get(g:, 'ale_r_lintr_options', 'lintr::with_defaults(object_usage_linter = NULL)')
+
 function! ale_linters#r#lintr#GetCommand(buffer) abort
     return ale#path#BufferCdString(a:buffer)
-    \   . 'Rscript -e ' . ale#Escape('lintr::lint(commandArgs(TRUE))') . ' %t'
+    \   . 'Rscript -e ' . ale#Escape('lintr::lint(commandArgs(TRUE)[1], eval(parse(text = commandArgs(TRUE)[2])))') . ' %t' . ' ' . ale#Escape(ale#Var(a:buffer, 'r_lintr_options'))
```
Comment (w0rp) [inline ale_linters/r/lintr.vim]: You can break this into some more lines with `\` at the start of the lines.

**Comment by w0rp** [comment]
I added a test for this, documented the new option, and I have merged this to master now.

**Comment by AlexAxthelm** [comment]
Thank you! I'm sorry that I never got back to this.

**Comment by w0rp** [comment]
No problemo!

**Pull request closed without merging**